.travis.yml

```diff
@@ -3,7 +3,9 @@ sudo: false
 cache: bundler
 
 rvm:
-  - 2.3.0
+  - 2.3.3
+  - 2.4.1
 
 env:
-  - RAILS_VERSION=5.0
+  - RAILS_VERSION=5.0.3
+  - RAILS_VERSION=5.1
```

Comment:
There will be changes to merge from latest master, when you rebase you'll see them.